### PR TITLE
CONN-1093:Add AdminGUI module in CONNECT WebLogic Ear

### PR DESCRIPTION
**Pull request by alameluchidambaram**

### CODE REVIEW

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -45,6 +45,17 @@
         <param-value>Production</param-value>
     </context-param>
 
+    <context-param>
+        <param-name>javax.faces.STATE_SAVING_METHOD</param-name>
+        <param-value>client</param-value>
```
Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/WEB-INF/web.xml]: This was initially "server", what does the change to "client" do?

File: Product/Production/Deploy/ear/weblogic/src/main/application/META-INF/weblogic-application.xml [hunk 2]
```diff
@@ -1,32 +1,41 @@
...
+        <package-name>org.bouncycastle.*</package-name>
+        <package-name>javax.faces.*</package-name>
+        <package-name>com.sun.faces.*</package-name>
+        <package-name>com.bea.faces.*</package-name>
+    </prefer-application-packages>
+    <prefer-application-resources>
+        <resource-name>javax.faces.*</resource-name>
+        <resource-name>com.sun.faces.*</resource-name>
+        <resource-name>com.bea.faces.*</resource-name>
+        <resource-name>META-INF/services/javax.servlet.ServletContainerInitializer</resource-name>
```
Comment (christophermay07) [inline Product/Production/Deploy/ear/weblogic/src/main/application/META-INF/weblogic-application.xml:40]: What is this resource?

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/WEB-INF/web.xml (hunk 1 above)
Comment (alameluchidambaram) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/WEB-INF/web.xml]: If param value is "client" then client cache the data and going to change it to default "server" for now.

File: Product/Production/Deploy/ear/weblogic/src/main/application/META-INF/weblogic-application.xml (hunk 2 above)
Comment (alameluchidambaram) [inline Product/Production/Deploy/ear/weblogic/src/main/application/META-INF/weblogic-application.xml:40]: This resource is defined to override the default ServletContainerInitializer and use the one that's within the application.

**Pull request closed without merging**